bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/BmpStringValue.java

```diff
@@ -59,6 +59,11 @@ public String stringValue() {
         return value;
     }
 
+    @Override
+    public StringValue bStringValue() {
+        return null;
```

Comment:
'null' return??

Reply:
Created an issue to track this: https://github.com/ballerina-platform/ballerina-lang/issues/20886 Need to migrate all implementations of BValue. Currently it returns null.